pageserver/src/tenant/config.rs

```diff
@@ -42,6 +42,12 @@ pub mod defaults {
     pub const DEFAULT_WALRECEIVER_LAGGING_WAL_TIMEOUT: &str = "3 seconds";
     pub const DEFAULT_MAX_WALRECEIVER_LSN_WAL_LAG: u64 = 10 * 1024 * 1024;
     pub const DEFAULT_EVICTIONS_LOW_RESIDENCE_DURATION_METRIC_THRESHOLD: &str = "24 hour";
+    // This parameter limits amount of image layer generated due to GC request.
+    // It should avoid storage space explosion caused by taken in account GC feedback for existed projects.
+    // Total amount of forced image layers is limited by forced_image_creation_limit*compaction_target_size
+    // per GC iteration. As far as GC will be able to remove old layers only after PiTR interval expiration,
```

Comment:
`per iteration` is a complicated way to do this. Now it all depends on scheduling. If we manually request GC, or if we change GC period, the behavior will change. GC should be an idempotent operation.

One solution is to make it on/off instead of a limit. This would also enable us to test the hypothesis that "this will decrease amplification afte some amount of time", bu fully turning it on for a few tenants. If we only use a limit, we'll have to wait months to evaluate this strategy that might not work.